### PR TITLE
Load intial account balances from blockchain and refactor loader flags

### PR DESCRIPTION
**Pull request by b-mueller**
- Adds new function `read_balance(address` to dynloader
- Account constructor now reads intial account balance from blockchain if dynloader is available
- Minor: Don't pass dynloader to `CREATOR_ACCOUNT` and `ATTACKER_ACCOUNT` which are virtual
- Use a single boolean / flag to determine if data should be loaded instead of separating code/storage/balances:

```
  --no-onchain-data. Don't attempt to retrieve contract code, variables and balances from the blockchain
```

### CODE REVIEW

File: mythril/support/loader.py [hunk 1]
```diff
@@ -47,6 +47,22 @@ def read_storage(self, contract_address: str, index: int) -> str:
...
         )
 
+    @functools.lru_cache(LRU_CACHE_SIZE)
+    def read_balance(self, address: str) -> str:
+        """
+
+        :param address:
+        :return:
+        """
+        if not self.storage_loading:
```
Comment (norhh) [inline mythril/support/loader.py]: we should change this variable to `onchain_loading` as this isn't storage right?

Comment (b-mueller) [reply mythril/support/loader.py]: I simplified this to a single boolean (see updated PR)

File: mythril/support/loader.py [hunk 2]
```diff
@@ -36,25 +32,39 @@ def read_storage(self, contract_address: str, index: int) -> str:
...
+    @functools.lru_cache(LRU_CACHE_SIZE)
+    def read_balance(self, address: str) -> str:
+        """
+
+        :param address:
+        :return:
+        """
+        if not self.active:
+            raise ValueError(
+                "Cannot load from the storage when the storage_loading flag is false"
```
Comment (norhh) [inline mythril/support/loader.py]:
```suggestion
                "Cannot load from storage when the loader is disabled"
```

File: mythril/support/loader.py [hunk 3]
```diff
@@ -36,25 +32,39 @@ def read_storage(self, contract_address: str, index: int) -> str:
...
+
+        :param address:
+        :return:
+        """
+        if not self.active:
+            raise ValueError(
+                "Cannot load from storage when the loader is disabled"
+            )
+        if not self.eth:
+            raise ValueError("Cannot load from the storage when eth is None")
```
Comment (norhh) [inline mythril/support/loader.py]:
```suggestion
            raise ValueError("Cannot load from the chain when eth is None")
```

**Pull request merged**